crates/rpc/src/jsonrpc/router.rs

```diff
@@ -116,7 +116,9 @@ pub async fn rpc_handler(
     body: axum::body::Bytes,
 ) -> impl axum::response::IntoResponse {
     // Only json content allowed.
-    if content_type != ContentType::json() {
+    if content_type != ContentType::json()
+        && content_type.to_string().as_str() != "application/json; charset=utf-8"
```

Comment:
Can't we parse the MIME thingy? It seems that's what the JSON extractor does in axum: https://docs.rs/axum/latest/src/axum/json.rs.html#147

Reply:
That's probably a better approach yeah. Will try that instead.